catalog/app/containers/Auth/saga.js

```diff
@@ -71,6 +71,10 @@ function* signUp(credentials) {
       if (e.status === 409 && e.json && e.json.message === 'Email already taken.') {
         throw new errors.EmailTaken({ originalError: e })
       }
+      if (HTTPError.is(e, 401, /domain \S+ is not allowed/i)) {
```

Comment:
I could just return an error code from the server, the auth domain, etc.

Reply:
yep, you can, not a big deal tho